apps/api/src/domain/models/playlist/commands/create-playlist.command.ts

```diff
@@ -4,19 +4,23 @@ import { ExternalEnum, NestedDataType, NonEmptyString, UUID } from '@coscrad/dat
 import { AggregateCompositeIdentifier } from '../../../types/AggregateCompositeIdentifier';
 
 export class PlayListCompositeId {
-    /**
-     * This is a bit of a hack. It circumvents our `CoscradDataTypes` and may
-     * cause problems for
-     * - Schema management
-     * - Anyone using our API directly (not via front-end)
-     *
-     * The simple answer is that you always have to tack on an
-     * `aggregateCompositeIdentifier`.
-     */
-    @NonEmptyString({
-        label: 'type',
-        description: 'song',
-    })
+    // TODO Should we have an @FixedValue decorator instead?
```

Comment:
This is a tough one. It never surfaces to end-users using the UX, but it's still important because it's possible to drive the back-end over the API and we can't ever trust the front-end to give us the right types here.

Reply:
Please link this comment as a comment to the [following story](https://www.pivotaltracker.com/story/show/185297262). Also please add a TODO with a link to the story in the code at this point.

Reply:
done